testdata/p4_16_errors/issue407-1.p4

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
deleting this whole file seems excessive. You should probably keep the negative tests that should still fail.

Reply:
Agree.

Reply:
You agree, but the file is still deleted.

Reply:
I will add this file and other files in the 'outputs' directory as well.